### PR TITLE
feat(ai): improve analytics report

### PR DESCRIPTION
**Pull request by AntoineJac**
### Summary

This PR will add new AI analytics into the PING anonymous report

### Checklist

- [x] The Pull Request has tests
- [x] A changelog file has been created under `changelog/unreleased/kong` or `skip-changelog` label added on PR if changelog is unnecessary. [README.md](https://github.com/Kong/gateway-changelog/blob/main/README.md)
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Issue reference

Fix #_[issue number]_

### CODE REVIEW

**Comment by hanshuebner** [comment]
Can you add tests, please?

File: spec/02-integration/22-ai_plugins/01-reports_spec.lua [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+          body = pl_file.read("spec/fixtures/ai-proxy/openai/llm-v1-chat/requests/good.json"),
+        })
+        assert.res_status(200, res2)
+
+        reports_send_ping(constants.REPORTS.STATS_TLS_PORT)
+
+        proxy_client:close()
+        proxy_client2:close()
+
+        ngx.sleep(0.1)
```
Comment (hanshuebner) [inline spec/02-integration/22-ai_plugins/01-reports_spec.lua:173]: `sleep` is an anti-pattern in tests.  Consider using `helpers.eventually`.

**Review by hanshuebner — CHANGES_REQUESTED** [review]
Thank you for adding the test!  The `sleep` needs to go, however, to avoid flakiness in high-load CI situations.

File: spec/02-integration/22-ai_plugins/01-reports_spec.lua (hunk 1 above)
Comment (AntoineJac) [reply spec/02-integration/22-ai_plugins/01-reports_spec.lua]: Thanks @hanshuebner 
agree and we actually don't need the sleep function so I removed it.

Do notice that the test will fail until https://github.com/Kong/kong/pull/12903 is merged. 

Thanks

File: spec/02-integration/22-ai_plugins/01-reports_spec.lua [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+local MOCK_PORT = helpers.get_available_port()
+
+for _, strategy in helpers.each_strategy() do
+  local admin_client
+  local dns_hostsfile
+  local reports_server
+
+  describe("anonymous reports for ai plugins #" .. strategy, function()
+    local reports_send_ping = function(port)
+      ngx.sleep(0.2) -- hand over the CPU so other threads can do work (processing the sent data)
```
Comment (jschmid1) [inline spec/02-integration/22-ai_plugins/01-reports_spec.lua:16]: Use the `eventually` helper to avoid bare sleeps and flakyness

**Review by jschmid1 — CHANGES_REQUESTED** [review]

File: spec/02-integration/22-ai_plugins/01-reports_spec.lua (hunk 2 above)
Comment (AntoineJac) [reply spec/02-integration/22-ai_plugins/01-reports_spec.lua]: Done. Thanks

**Review by jschmid1 — APPROVED** [review]
Approved, granted tests pass.

**Review by hanshuebner — APPROVED** [review]

**Pull request merged**